pkg/ccl/changefeedccl/changefeed_stmt.go

```diff
@@ -876,6 +876,14 @@ func validateDetails(details jobspb.ChangefeedDetails) (jobspb.ChangefeedDetails
 			)
 		}
 	}
+	{
+		_, isSet := details.Opts[changefeedbase.OptPrimaryKeyFilter]
+		if isSet {
+			return jobspb.ChangefeedDetails{}, errors.Errorf(
```

Comment:
maybe set a feature not supported pgcode?

Reply:
done.